docs/_docs/snapshots/snapshots.adoc

```diff
@@ -178,9 +172,59 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 |N > M | Unsupported.
 |===
 
+=== Automatic snapshot restore procedure
+
+The automatic restore procedure allows the user to restore cache groups from a snapshot on an active cluster using the Java API or link:tools/control-script[command line script].
```

Comment:
Avoid "using" in docs, favor "that uses" or "by using" to avoid confusion.

The automatic restore procedure allows the user to restore cache groups from a snapshot on an active cluster by using the Java API or link:tools/control-script[command line script].

Reply:
Done, thanks